sale_exception/views/sale_view.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" ?>
+<odoo>
+        <record id="action_sale_test_tree" model="ir.actions.act_window">
+                  <field name="name">Sale Exception Rules</field>
+                  <field name="res_model">exception.rule</field>
+                  <field name="view_type">form</field>
+                  <field name="view_mode">tree,form</field>
+                  <field name="view_id" ref="base_exception.view_exception_rule_tree"/>
+                  <field name="domain">[('rule_group', '=', 'sale')]</field>
+                  <field name="context">{'active_test': False, 'default_rule_group' : 'sale'}</field>
+              </record>
+
+        <menuitem
+                action="action_sale_test_tree"
+                id="menu_sale_test"
+                sequence="90"
+                parent="sale.menu_sales_config"
+        />
+
+        <record id="view_order_form" model="ir.ui.view">
+            <field name="name">sale_exception.view_order_form</field>
+            <field name="model">sale.order</field>
+            <field name="inherit_id" ref="sale.view_order_form"/>
+            <field name="arch" type="xml">
+                <field name="name" position="after">
+                    <group>
+                    <field name="main_exception_id" options='{"no_open": True}'
+                           class="oe_inline" string="Error:"
+                           attrs="{'invisible':[('main_exception_id','=', False)]}"/>
+                    </group>
+                </field>
+                <xpath expr="//field[@name='date_order']/.." position="inside">
+                    <field name="ignore_exception" states="sale" />
+                </xpath>
+                <xpath expr="//group[@name='sales_person']/.."
+                        position="inside">
+                    <newline />
+                    <group name="exception" colspan="2" col="2">
+                        <separator string="Exception" colspan="2"/>
+                        <field name="exception_ids" colspan="2" nolabel="1"/>
```

Comment:
I recommend defining a tree view inside this field to show only a few fields of interest.

Specifically, my concern is the 'active' field because it's easy to deactivate the rule by mistake from the sales order view:
![image](https://user-images.githubusercontent.com/1914185/55802461-958d1e00-5ad8-11e9-876f-bc20beff7109.png)

I would just show `name` and `description` here.